server/auth/sso/sso.go

```diff
@@ -28,11 +28,13 @@ import (
 const (
 	Prefix                              = "Bearer v2:"
 	issuer                              = "argo-server"                // the JWT issuer
-	expiry                              = 10 * time.Hour               // how long JWT are valid for
 	secretName                          = "sso"                        // where we store SSO secret
 	cookieEncryptionPrivateKeySecretKey = "cookieEncryptionPrivateKey" // the key name for the private key in the secret
 )
 
+// how long JWT are valid for
+var expiry time.Duration
```

Comment:
I think we should remove this global variable - instead, add a field called `expiry` to the struct `sso`